pulsar-client/src/main/java/org/apache/pulsar/client/impl/ConsumerImpl.java

```diff
@@ -2164,7 +2161,7 @@ private CompletableFuture<Void> seekAsyncInternal(long requestId, ByteBuf seek,
         if (!duringSeek.compareAndSet(false, true)) {
             log.warn("[{}][{}] Attempting to seek operation that is already in progress, cancelling {}",
                     topic, subscription, seekBy);
-            seekFuture.cancel(true);
+            seekFuture.completeExceptionally(new IllegalStateException());
```

Comment:
`new IllegalStateException()` -> `new IllegalStateException("String.format("%s,%s attempting to seek operation that is already in progress", topic, subscription))`  may be better  ?
@michaeljmarshall

Reply:
Thank you! I'll factor out a `message` variable and use it for log message and error message.